app/src/main/java/com/example/voyagerx/ui/fragments/editprofile/EditProfileFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.voyagerx.ui.fragments.editprofile
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.voyagerx.R
+import com.example.voyagerx.databinding.FragmentEditProfileBinding
+import com.example.voyagerx.ui.fragments.userprofileac.ProfileFragment
+
+class EditProfileFragment : Fragment() {
+    private lateinit var binding: FragmentEditProfileBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        binding = FragmentEditProfileBinding.inflate(inflater, container, false)
+
+        settingButtonListeners()
+        grabCurrentInfoFromDatabase()
```

Comment:
A minor suggestion: For consistency, I think it would be helpful to keep verbs in the same tense. So it would be `setButtonListeners()` and `grabCurrentInfoFromDatabase()`

Reply:
Good point. Its good practice to try and name functions and variables as we would in a large-scale professional project. On top of Celinas suggestion for using "setButtonListeners", here are two things here from me:

1. "Grab" is synonymous with "Get" and "get methods", which are already a thing in software programming. Easy change.
2. "info" is vague and broad. You dont want teammates and future devs to ever wonder..."what kind of info?"

aka: something like, "getCurrentLaunchFromDatabase()" 

Do you think that function name would be more accurate?

@Celinarabe & @brownyte  , thoughts or suggestions?

Reply:
Yep I agree with that!

Reply:
@Celinarabe & @joelc1225, I agree as well. I pushed a commit with both of your requested changes implemented. Thank you both for your time!